pkgs/by-name/tc/tcount/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tcount";
+  version = "0-unstable-2023-04-20";
+
+  src = fetchFromGitHub {
+    owner = "rrethy";
+    repo = "tcount";
+    rev = "341d9aa29512257bf7dfd7e843d02fdcfd583387";
+    hash = "sha256-M4EvaX9qDBYeapeegp6Ki7FJLFosVR1B42QRAh5Eugo=";
+  };
+
+  useFetchCargoVendor = true;
+
+  cargoHash = "sha256-Sn7hu+2jQFd2u8tpfTxnEO+syrO96gfgz6ouHxJnpLg=";
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    description = "Count your code by tokens and patterns in the syntax tree. A tokei/scc/cloc alternative";
+    homepage = "https://github.com/rrethy/tcount";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.unix;
```

https://github.com/RRethy/tcount/blob/341d9aa29512257bf7dfd7e843d02fdcfd583387/README.md?plain=1#L43

>Mac or Linux

They describes only supports Linux + MacOS
So excluding other Unix family might be stable?

```suggestion
    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

https://github.com/NixOS/nixpkgs/blob/960c72853a13924b2f966345ceb5c0951c4f0d55/lib/systems/doubles.nix#L111
https://github.com/NixOS/nixpkgs/blob/8f77bbf229564cdad33748822219e0fb1474e144/lib/systems/inspect.nix#L94